src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -645,6 +645,34 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP112",
                 $"The \"{LanguageConstants.TargetScopeKeyword}\" cannot be declared multiple times in one file.");
+
+            public Diagnostic InvalidModuleScopeForTenantScope() => new Diagnostic(
+                TextSpan,
+                DiagnosticLevel.Error,
+                "BCP113",
+                $"Unsupported scope for module deployment in a \"{LanguageConstants.TargetScopeTypeTenant}\" target scope. " +
+                $"Allowed scopes include current: tenant() or leave blank, named management group: managementGroup(<name>), or named subscription: subscription(<subId>).");

```

Comment:
I think "leave" makes it kind of weird. Should we drop that verb? Also blank could be misleading as well. Maybe omit that bit entirely?

Reply:
Reworded to make it a bit more readable.